File: SourceCode/XeKhachSpring/src/main/java/net/timxekhach/operation/data/entity/Employee.java
package net.timxekhach.operation.data.entity;
// ____________________ ::IMPORT_SEPARATOR:: ____________________ //
import lombok.Getter;
import lombok.Setter;
import javax.persistence.Entity;
import net.timxekhach.operation.data.mapped.Employee_MAPPED;
// ____________________ ::IMPORT_SEPARATOR:: ____________________ //

@Entity @Getter @Setter
public class Employee extends Employee_MAPPED {

    public Employee() {}
    public Employee(Company company, User user) {
        super(company, user);
    }

// ____________________ ::BODY_SEPARATOR:: ____________________ //
// ____________________ ::BODY_SEPARATOR:: ____________________ //

}

File: SourceCode/XeKhachAngular/src/app/business/entities/employee.ts
import {XeEntity} from "./xe-entity";
import {User} from "./user";
import {Company} from "./company";

export class Employee extends XeEntity {
  companyId: number;
  employeeId: number;
  userId: number;
  isLock: boolean;
  user: User;
  company: Company;

  constructor() {
    super();
  }
}
